app/models/concerns/host_common.rb

```diff
@@ -146,10 +146,12 @@ def crypt_root_pass
   end
 
   def param_true?(name)
+    Foreman::Deprecation.renderer_deprecation('1.16', __method__, 'host_param_true?')
```

Comment:
1.14 was branched last week, this should now be 1.17 as it's unlikely to be included in 1.14 now.